sdk/typescript/src/types/transactions.ts

```diff
@@ -237,6 +237,11 @@ export const SuiExecuteTransactionResponse = union([
       confirmed_local_execution: boolean(),
     }),
   }),
+  object({
+    certificate: CertifiedTransaction,
+    effects: SuiCertifiedTransactionEffects,
+    confirmed_local_execution: boolean(),
+  }),
```

Comment:
@666lcz  I didn't mark this one as deprecated but I expand the enum instead. All sdk e2e tests pass

Reply:
Nice! I think you can remove line 232 (which has been removed from the RPC side long ago). Could you also add a `// TODO: remove after devnet 0.25.0(or 0.24.0?) is released` before line 233? Thanks!